e2e/integration/lighthousetests.js

```diff
@@ -2,7 +2,7 @@ import { gaugechart } from "../helper/apps.list";
 
 context( 'Test lighthouse', () => {
     before( () => {
-        cy.visit( Cypress.env( 'STAGE_HOST' ) + 'lighthouse' );
+        cy.visit( Cypress.env( 'QA_HOST' ) + 'lighthouse' );
```

Comment:
Why the env varrs are changes i think the jenkins config should be changed also

Reply:
Hi @riginoommen , I had changed the env variable because test was failing on stage env but not on qa env. lighthouse was tested by me on qa env